### PR TITLE
#2804  Add reasonable defaults, disable submission.

### LINKED ISSUE #2804: backups shows scheduling in the ui before scheduling is defined in the api
The day of the week will return as Unscheduled (or similar) when the back-ups schedule is not set.

The UI is incorrectly showing a date and time that has not been set.

As a user who as enabled back-up for the first time the "Day of the Week" dropdown on `linodes/:id/backups/settings` should display "Choose a day"

### PR DESCRIPTION
**Pull request by rjstires**
Had to open a new branch and PR, somehow my original got tainted.

screen shot 2017-12-12 at 3 23 16 pm
screen shot 2017-12-12 at 3 23 24 pm

Closes #2804

### CODE REVIEW

File: src/linodes/linode/backups/components/ScheduleForm.js [hunk 1]
```diff
@@ -111,13 +128,13 @@ export class ScheduleForm extends Component {
...
               name="window"
-              value={window || ''}
+              value={isScheduled ? window : ''}
               onChange={this.onChange}
               options={adjustedScheduleOptions}
             />
-            <small className="form-text text-muted">
-            Daily backups will be attempted {start.format('HH:mm z')} - {finish.format('HH:mm z')}.
-            </small>
+            {isScheduled && <small className="form-text text-muted">
```
Comment (displague) [inline src/linodes/linode/backups/components/ScheduleForm.js]: I might have gone with a more obvious `isScheduled ? <small /> : ''` or `! isScheduled ? '' : <small />` (which matches the general pattern in this repo).

`isScheduled && <small />` works in the truth state, and likewise jsx renders an empty string when in the falsey state that `{isScheduled}` can provide.

File: src/linodes/linode/backups/components/ScheduleForm.js [hunk 2]
```diff
@@ -64,7 +66,11 @@ export function createAdjustedScheduleOptions(timezone, day) {
     };
   });
 
-  return sortBy(list, (i) => i.label);
+  list = sortBy(list, (i) => i.label);
+
+  list.unshift({ label: 'Choose a day', value: '' });
```
Comment (displague) [inline src/linodes/linode/backups/components/ScheduleForm.js]: These are times, not days.

File: src/linodes/linode/backups/components/ScheduleForm.js [hunk 3]
```diff
@@ -16,6 +17,7 @@ import api from '~/api';
 import { dispatchOrStoreErrors } from '~/api/util';
 
 const dayOptions = [
+  { value: 'Scheduling', label: 'Choose a time' },
```
Comment (displague) [inline src/linodes/linode/backups/components/ScheduleForm.js]: These are days, not times.

**Comment by displague** [comment]
When you change the date to a valid choice, then change the time back to "Choose A Time":
> TypeError: Cannot read property 'format' of undefined

A different bug exists when trying to change the time before the date has been chosen, preventing a time selection from being made.

**Comment by rjstires** [comment]
Originally I had considered disabling time of day before day of week was set, think that's a reasonable idea?

**Comment by asauber** [comment]
LGTM

**Pull request merged**